src/main/java/com/hartwig/platinum/kubernetes/SampleArgument.java

```diff
@@ -15,28 +13,12 @@ public interface SampleArgument {
 
     Map<String, String> arguments();
 
-    static SampleArgument sampleJson(final TumorNormalPair pair, final String runName) {
-        return ImmutableSampleArgument.builder()
-                .id(pair.name().toLowerCase())
-                .putArguments("-sample_json", format("samples/%s-%s", pair.name().toLowerCase(), runName))
-                .putArguments("-set_id", pair.name())
-                .putArguments("-run_id", runName + pair.tumorIndex().map(ti -> "-" + ti).orElse(""))
-                .build();
-    }
-
-    static SampleArgument biopsy(final String biopsy, final String runName) {
-        return ImmutableSampleArgument.builder()
-                .id(biopsy.toLowerCase())
-                .putArguments("-biopsy", biopsy)
-                .putArguments("-run_id", runName)
-                .build();
-    }
-
-    static SampleArgument runId(final String biopsy, final Long runId) {
+    static SampleArgument sampleJson(final PipelineInput pair, final String runName) {
```

Comment:
I like the simplification here! Maybe we can change the call time of this method. And change the signature of the KubernetesCluster to `KubernetesCluster.submit(final List<PipelineInput> pairs)`

Reply:
Even simpler, we can just remove the argument altogether, since it is passed into `kubernetesEngine.findOrCreate` already.